components/CreateStories.tsx

```diff
@@ -0,0 +1,30 @@
+import { StoryblokComponent, type ISbStoryData } from '@storyblok/react/rsc';
+
+/**
+ * Use this component with the references (or the multiple/single option) field type
+ * when they reference an instance of a story, e.g., the masthead picker in Basic Page.
+ * The try/catch was added since we added it on SAA Homesite here as well:
+ * https://github.com/SU-SWS/saa_alumni/pull/563
+ */
+
+type CreateStoriesProps = {
+  stories: ISbStoryData[];
+  [key: string]: any;
+};
+
+export const CreateStories = ({ stories, ...props }: CreateStoriesProps) => {
+  let currentStory;
+  if (stories) {
+    try {
+      return stories.map((story) => {
+        currentStory = story;
+        return <StoryblokComponent key={story.content._uid} blok={story.content} {...props} />;
+      });
+    } catch (error) {
+      console.error('Could not create story', currentStory);
+    }
+  }
+
+  // Return null if no content provided.
+  return null;
+};
```

Comment:
Reusing this from our other Storyblok sites, with some updates with TypeScript and using `StoryblokComponent`

Reply:
Neat.